tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Datadog.Trace.Tools.Runner.IntegrationTests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Platforms>x64;x86</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Condition="$(TargetFramework.StartsWith('net4')) == false" Include="..\..\src\Datadog.Trace.Tools.Runner\Datadog.Trace.Tools.Runner.Tool.csproj" />
```

Comment:
That's not great, but the CI tries to build all the test projects for net461, even though this one is not compatible. After some trial and error, I figured out it was much easier to keep the net461 target for this project (even if it actually produces no tests) than excluding this project in Nuke only for net461.